docs/Researcher/cli-reference/runai-submit.md

```diff
@@ -368,10 +372,25 @@ runai submit --job-name-prefix -i gcr.io/run-ai-demo/quickstart -g 1
 
 > External access type to interactive jobs. Options are:
 >
->  * `portforward` (deprecated)
->  * `loadbalancer`
 >  * `nodeport`
 >  * `external-url`
+>
+> This flag can be used both in the standard format which uses command flag for all the options.
+>
+> For example:
+>
+> `runai submit test-jup -p team-a -i gcr.io/run-ai-demo/jupyter-tensorboard --service-type external-url --port 8888`
+>
+> `runai submit test-np -p team-a -i ubuntu --service-type nodeport --port 30000:7070`
+>
+> However, you can also use the newer format, where the options are listed in CSV style using multiple instances of the same option and commas to seperate the values for them.
```

Comment:
in a new line, I'd explain that if one wants to use more than one service, he/she has to use the CSV format. this is the essence of this change

Reply:
I revered the order and corrected the grammar issue you found, along with a spelling mistake we both missed.